src/v2/Apps/BuyerGuarantee/buyerGuaranteeRoutes.tsx

```diff
@@ -30,8 +30,17 @@ export const buyerGuaranteeRoutes: RouteConfig[] = [
         },
         query: graphql`
           query buyerGuaranteeRoutes_BuyerGuaranteeQuery {
-            buyerGuarantee: me {
-              ...BuyerGuaranteeIndex_buyerGuarantee
+            headerImage: artwork(id: "5dd8084d257aaf000e4a0396") {
```

Comment:
These artworks might get deleted or the id might change breaking this page. I think it is better to either use static images. Then there would be no need for relay for these pages I think.